### PR TITLE
Set console logo with css instead of within markup so that it can be …

### PR DESCRIPTION
**Pull request by sg00dwin**
…customized. 

ref: https://github.com/openshift/origin/issues/4148

### CODE REVIEW

File: assets/app/styles/_core.less [hunk 1]
```diff
@@ -39,6 +39,14 @@ html, body {
   .navbar-pf {
     background: @navbar-os-bg-color;
     .navbar-header {
+      .navbar-brand {
+        .brand-logo {
+          background: url('/images/logo-origin-thin.svg');
```
Comment (liggitt) [inline assets/app/styles/_core.less]: absolute URL won't work...

**Comment by jwforres** [comment]
@brenton FYI the override commit for enterprise to change the logo will need to get updated

[merge]

**Comment by jwforres** [comment]
@sg00dwin needs bindata

File: assets/app/styles/_core.less (hunk 1 above)
Comment (spadgett) [inline assets/app/styles/_core.less]: @sg00dwin I believe the right relative path is

```
background: url('../images/logo-origin-thin.svg');
```

**Comment by brenton** [comment]
@sdodson, watch out for this the next time you pull from origin.  We'll need to make sure we don't wipe out the OSE logo.

**Comment by sg00dwin** [comment]
@jwforres updated with relative path and bindata

**Comment by jwforres** [comment]
[merge]

**Pull request merged**